src/intervals/set_operations.jl

```diff
@@ -131,7 +133,7 @@ union(a::Interval, b::Interval) = hull(a, b)
 #
 # union(a::Interval, b::Interval) = union(promote(a, b)...)
 union(a::Complex{<:Interval},b::Complex{<:Interval}) = hull(a, b)
-
+union(a, b, c...) = reduce(union, (a, b, c...))
```

Comment:
Is there a reason for not using  a... here?

Reply:
No, no reason... I've changed it, and now it uses `reduce`.